lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3061,13 +3079,18 @@ LogicalResult ConvertAtenOp<AtenSliceTensorOp>::matchAndRewrite(
   if (!matchPattern(op.getStart(), m_TorchConstantInt(&start)))
     return rewriter.notifyMatchFailure(op, "start must be a Scalar constant");
 
-  if (start < 0)
-    return rewriter.notifyMatchFailure(op, "Currently unsupported: start < 0");
+  if (start < 0) {
+    start = start + selfType.getShape()[dim];
```

Comment:
You need to check that it is positive after this

Reply:
Done